org.lflang/src/org/lflang/cli/Lfc.java

```diff
@@ -34,7 +31,14 @@
  *
  * @author Marten Lohstroh
  * @author Christian Menard
+ * @author Atharva Patil
  */
+@Command(
+    name = "lfc",
+    // Enable usageHelp (--help) and versionHelp (--version) options.
+    mixinStandardHelpOptions = true,
+    // TODO: Import version from StringsBundle.properties. 
+    version = "lfc 0.3.1-SNAPSHOT")
```

Comment:
This should use `LocalStrings.VERSION` to retrieve the current version.

Reply:
Done!